src/Goerli-DssSpellCollateral.sol

```diff
@@ -30,58 +68,180 @@ contract DssSpellCollateralAction {
     //
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
-    //
+
+    uint256 constant ZERO_PCT_RATE                  = 1000000000000000000000000000;
+
+    // --- Math ---
+    uint256 public constant WAD = 10**18;
+    uint256 public constant RAY = 10**27;
+    uint256 public constant RAD = 10**45;
+
+    // -- RWA007 MIP21 components --
+    address constant RWA007                         = 0x02091C59C92fc5E0575b3B133c3caa4f57D79b0e;
+    address constant MCD_JOIN_RWA007_A              = 0xe3BA7B6c922cb4622AfF00C2e2A467fF6A06CA29;
+    address constant RWA007_A_URN                   = 0x78D9ccbe652E2aE7B11289B4F202278058aEfeeA;
+    address constant RWA007_A_JAR                   = 0xcF8A6445B6e22A8e2Fe9d7bd21410bB7a8b8D054;
+    // Goerli: Coinbase / Mainnet: Coinbase
+    address constant RWA007_A_OUTPUT_CONDUIT        = 0x9B4Af5F55b23cC2D86f58d7c08cD025bA1901013;
+    // Jar and URN Input Conduits
+    address constant RWA007_A_INPUT_CONDUIT_URN     = 0xFC80d4037dDadDb380e752AE46Aca262eEb812fC;
+    address constant RWA007_A_INPUT_CONDUIT_JAR     = 0x9ca5F7b1F5DCC287657c20547176a6733EB2c046;
```

Comment:
these contract deployments would need to be replaced with new ones once https://github.com/makerdao/mip21-toolkit/pull/21 is finalized, approved and merged.

Reply:
@naszam can you please also review and approve https://github.com/makerdao/mip21-toolkit/pull/21

Reply:
@nduchak we're waiting for `rwa007` contract redeployments to proceed with reviews, two approvals should be sufficient to merge and proceed, you might want to check with @godsflaw for final signoff though, on latest changes, if you need.